dashboard/test/factories/pd_factories.rb

```diff
@@ -916,7 +916,7 @@
       contact_phone "1597534862"
       contact_relationship "it's complicated"
       dietary_needs "Food Allergy"
-      dietary_needs_food_allergy_details "memories"
+      dietary_needs_details "memories"
```

Comment:
This was erroneously named. Apparently never used this trait, or at least never validated, because it was failing validation.